packages/stream_video/lib/src/call/state/mixins/state_coordinator_mixin.dart

```diff
@@ -498,6 +506,7 @@ mixin StateCoordinatorMixin on StateNotifier<CallState> {
           return member;
         }
       }).toList(),
+      capabilitiesByRole: capabilitiesByRole,
```

Comment:
In `coordinatorCallPermissionsUpdated` you check for the current user id. Are we certain that `StreamCallMemberUpdatedPermissionEvent` can only be for the current user?

Reply:
This event updates the `members` list and `capabilitiesByRole` map, and those are global for the call. It does not update anything specific to the single user.